src/org/labkey/test/tests/list/CrossFolderListTest.java

```diff
@@ -49,7 +55,94 @@ private void doSetup()
+        // expect the UI to refuse
+        var listEditPage = new ListHelper(getDriver()).beginCreateList(SUBFOLDER_A_PATH, listName);
+        listEditPage.addField(new FieldDefinition("field", FieldDefinition.ColumnType.String));
+        var errors = listEditPage.clickSaveExpectingErrors();
+
+        assertThat(errors).as("error should explain that our list name is already in use")
+                .containsOnly("The name 'name_collision_test_list' is already in use.");
+    }
+
+    @Test
+    public void testListViewShowsContainerByDefault()
+    {
+        goToProjectHome(getProjectName());
+        var listsPage = goToManageLists();
+
+        // ensure 'folder' is represented in the default view here
+        assertThat(listsPage.getGrid().getColumnNames()).as("expect default view to contain 'Container'")
+                .contains("Container");
+
+        // expect default folder filter to be "current folder, project, and shared project"
+        assertThat(listsPage.getGrid().getContainerFilter()).as("expect current, project, shared")
+                .isEqualTo("Current folder, project, and Shared project");
+    }
+
+    @Test
+    public void testCanDeleteTestInSeparateFolder() throws Exception
+    {
+        // create a list in a subfolder
+        String listName = "suba_folder_delete_test";
+        ListDefinition listDef = createListDef(listName, testFields());
+        listDef.create(createDefaultConnection(), SUBFOLDER_A_PATH);
+
+        // also a list in '/Shared'
+        ListDefinition sharedListDef = createListDef(SHARED_LIST, testFields());
+        sharedListDef.create(createDefaultConnection(), "/Shared");
+
+        goToProjectHome();
+        var helper = new ListHelper(getDriver());
+        helper.beginAtList(getProjectName(), SHARED_LIST);
+
+        // insert a record into the shared list, in the current folder
+        DataRegionTable.DataRegion(getDriver()).find().clickInsertNewRow()
+                .update(Map.of("intColumn", 1, "decimalColumn", "2.2", "stringColumn", "stringy"));
+
+        // navigate to the shared list but view it in the project
+        helper.beginAtList(getProjectName(), SHARED_LIST);
+
+        // click the 'stringy' link in the record we put into the shared list
+        clickAndWait(Locator.linkWithText("stringy"));  // the presence of this link confirms data in this view
+        assertThat(getURL().toString()).as("expect data url to be in project, is not in /Shared")
+                .contains("CrossFolderListTest")
+                .doesNotContain("Shared");
+        // now click on the list link, expecting that to navigate us to where it is defined, in ths /Shared folder
+        clickAndWait(Locator.linkWithText(SHARED_LIST));
+        assertThat(getURL().toString()).as("expect link to shared list to navigate to shared folder")
+                .doesNotContain("CrossFolderListTest")
+                .contains("Shared");
```

Comment:
Should be able to do an exact match on the container path. `getCurrentContainerPath` includes a leading slash but that's easy enough to handle:
```suggestion
        assertThat(getCurrentContainerPath()).as("expect data url to be in project, is not in /Shared")
                . isEqualTo("/" + getProjectName());
        // now click on the list link, expecting that to navigate us to where it is defined, in the /Shared folder
        clickAndWait(Locator.linkWithText(SHARED_LIST));
        assertThat(getCurrentContainerPath()).as("expect link to shared list to navigate to shared folder")
                .isEqualTo("/Shared");
```